main.js

```diff
@@ -1 +1,115 @@
-console.log("Tic Tac Toe")
\ No newline at end of file
+// alternatives for var?
+var grid, playerActive, gameGoing, counts;
+
+function makeArray(w, h, val) {
```

Comment:
Name your functions so that other coders can immediately know what this function does without going through the code

Call it `createGameboard`

Reply:
The function should not accept unnecessary parameter. For example, in your case, the first and second parameters make sense because they control the size of the gameboard, but the last value is not necessary for the function consumer to concern at all. Remember, encapsulation? A good function should be well encapsulated and hiding all the details